lib/build_cache.py

```diff
@@ -472,6 +478,11 @@ def git_restore(self, quick):
          ch.ossafe(os.mkfifo, "can’t make FIFO: %s" % self.path, self.path_abs)
       elif (self.path.git_incompatible_p):
          self.path_abs.git_escaped.rename_(self.path_abs)
+      # Restore extended attributes
+      if ch.xattrs:
```

Comment:
I wonder if it would be good to aim to make this unconditional in the future? In which case we'd need an issue.